torchtune/modules/tokenizers/_tiktoken.py

```diff
@@ -335,6 +337,7 @@ def tokenize_messages(
         messages: List[Message],
         max_seq_len: Optional[int] = None,
         tokenize_header: bool = True,
+        add_eos: bool = True,
```

Comment:
So this was defaulted to true before?

Reply:
Since generate recipe wasn't using tokenize_messages, it wasn't a problem before. As this was only used for training before

Reply:
Since generate recipe wasn't using tokenize_messages, it wasn't a problem before. As this was only used for training before